ingest/scripts/compare_hashes.py

```diff
@@ -185,6 +186,7 @@ def main(
 
     submitted: dict = json.load(open(old_hashes, encoding="utf-8"))
     ingested_insdc_accessions = get_approved_submitted_accessions(submitted)
+    reingested_insdc_accessions = set()
```

Comment:
To understand, does this store just reingested sequences or all sequences ingested this time?(including new ones)

Reply:
including new ones :-)

Reply:
So to me this is odd (inaccurate) variable naming, I would call this variable `ingested_insdc_accessions`, or `current_ingested_accessions` and the variable above one `already_ingested_accessions` or something. (But not blocking, just since we already started discussing it)